src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -344,6 +350,7 @@ public function addPaymentAction(int $orderId, Request $request): RedirectRespon
 
                 $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
             } catch (Exception $e) {
+                throw $e;
```

Comment:
Debug only

Reply:
Wooops, that's why I raised #15732 issue.

Reply:
:trollface: are you a poker player ?